LR_Tourist/TouristWebApp/TouristWebApp.csproj

```diff
@@ -6,7 +6,10 @@
 
   <ItemGroup>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.1">
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="3.1.3" />
```

Comment:
After moving DbContext for Identity to the DAL project - these libs can be removed.

Reply:
fixed)